catalog-info.yaml

```diff
@@ -12,6 +12,10 @@ spec:
   owner: group:ingest-fp
   system: platform-ingest
   lifecycle: production
+  dependsOn:
+    - "component:package-registry"
+    - "resource:buildkite-pipeline-release-package-registry-distribution"
+
```

Comment:
This dependencies have been added to the Component resource. 
It looks like it is where it is commonly defined.

More info: https://backstage.io/docs/features/software-catalog/well-known-relations/#dependson-and-dependencyof